src/adagrams.js

```diff
@@ -1,15 +1,153 @@
+/*constructing a leeterpool object with letter as key and no. of times it can be drawn as value*/
+const LETTERPOOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const drawnLetters = [];
+  //Evaluating total size of probable letters can be drawn
+  let totalSize = Object.values(LETTERPOOL).reduce((acc, val) => acc + val, 0);
+  let letterPoolArr = Object.entries(LETTERPOOL);
+  let count = 0,
+    letterIdx;
```

Comment:
My preference is that each declaration to be on its own line, you can read more about it [here](https://stackoverflow.com/a/7384541)

```suggestion
  let count = 0;
  let letterIdx;
```